precompiles/nfts/swap/src/tests.rs

```diff
@@ -17,13 +17,43 @@
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
 use crate::{mock::*, NftsSwapPrecompileCall};
+use pallet_evm_precompile_nfts_tests::{ExtBuilder, ALICE, BOB};
+use precompile_utils::testing::*;
+use sp_core::U256;
+use pallet_evm_precompile_nfts_types::solidity::{OptionalU256,OptionalPriceWithDirection, PriceWithDirection, PriceDirection};
+use pallet_nfts::Pallet as NftsPallet;
+use frame_support::assert_ok;
+use frame_system::pallet_prelude::OriginFor;
 
 type PCall = NftsSwapPrecompileCall<Runtime>;
 
 fn _precompiles() -> Precompiles<Runtime> {
```

Comment:
remove `_` prefix to the function name as it was just because it was unused to make the compiler happy